### ISSUE TITLE
Mod Breaking Lycintes Mobs

### ISSUE THREAD

**Issue by Shibva** [ArekkuusuJerii/BetterHurtTimer#13]
I noticed that ever since I added this mod that it messed with the defense attributes with lycanites mobs

usually the aegis has an immunity to high damage, however when this mod is present, its damage limit is messed with and allows me to one shot them

while this may not seem like an issue, it is for me for it breaks some of the defense attributes

can you please look into this?

**Comment by BunnyCinnamon** [comment]
Huh, I don't mess with damage.
what does your config look like and how do you kill them? Bow, Guns, Swords?

**Comment by Shibva** [comment]
not sure at this point, using melle weapons though

**Comment by BunnyCinnamon** [comment]
1.12.2? What sword are you using?

**Comment by Shibva** [comment]
just any meelee weapon in general and yes 1.12.2

**Comment by BunnyCinnamon** [comment]
Do you know the difference in damage with BHT on and off?

**Comment by BunnyCinnamon** [comment]
Also, aegis do not have a damage limit, just increased shield when their health is below 25%

**Comment by Shibva** [comment]
yes, the damage for the aegis is supposed to be half a heart when its "shields" are up

**Comment by BunnyCinnamon** [comment]
Hmmm but in the code it reads that it has a 50% chance of increased shield if after you hit if its health is under 25%.
Not that it will survive a fatal hit

**Comment by BunnyCinnamon** [comment]
oh wait were you replying to the first or second message

**Comment by Shibva** [comment]
this was at full health

look, idk but I removed the mod a While ago when Ifound out the Dependency for Ender skills
so idk 

Sorry

**Comment by BunnyCinnamon** [comment]
yeap idk

**Comment by BunnyCinnamon** [comment]
same

**Comment by Shibva** [comment]
all I Can conclude is that Better Hurt timer seemst o be bypassing the amor

While itself dosent apear to have an armor value (according to Wawla)
